examples/traefik/README.md

```diff
@@ -0,0 +1,14 @@
+# Basic configuration to use with the traefik reverse proxy
+
+Note: Tested with traefik 1.7
+
+- Remove the port-binds for the web service
+- The provided example assumes an external network with the name "web"
+- I've added comments where to adjust the docker-compose.yml, but I assume that you know your stuff, if you're running traefik.
```

Comment:
this is not appropriate for a documentation.

Reply:
I've assumed(wrongly?) that people who wants to run jitsi behind traefik, know a bit about setting it up and just added the steps which I've done. I can write a complete setup-guide including setting up traefik from the getgo, if you consider this more helpful. Not sure, if the traefik-setup-part would be out of scope?

Reply:
the point is that you're writing in the singular first person, unlike the rest of the document. also punctuation is missing, there are unnecessary line breaks in the markup. just use a hyperlink to reference traefik, don't point out the obvious ("you know your stuff, if …")

Reply:
yes, an introduction to traefik would be out of scope.